__tests__/package-json-versions/v11_1-all-override-with-unmatching.ts

```diff
@@ -32,20 +32,26 @@ export const packageJsonWith11_1AllWithoutNgPackagr = {
   },
   private: true,
   dependencies: {
+    unmatching: 'unmatching',
```

Comment:
Could you provide me some info about what should happen with this **unmatching** package? I don't identify if it will be deleted or ignored.

Reply:
It will remain untouched. Not easy to see with these large scope tests.